server/datastore/mysql/migrations/tables/20250318165922_AddChecksumAndSecretsToWindowsProfiles.go

```diff
@@ -0,0 +1,40 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250318165922, Down_20250318165922)
+}
+
+func Up_20250318165922(tx *sql.Tx) error {
```

Comment:
Just double-checking as it is already included in the PR checklist, but have you tested if/how this migration will any impact existing time-based comparisons related to profile delivery, reconciliation, verification, etc.? 

For example, in the `UPDATE` statement should we preserve the `hmwp.updated_at` timestamp in line 31? 

I don't recall offhand whether `ALTER` statements themselves causes `ON UPDATE` to fire so it may be worth a quick test to confirm that as well.